baklava/src/utils/rpc/ipc.ts

```diff
@@ -0,0 +1,57 @@
+import {
+    ipcMain,
+} from "electron";
+import { setPresence } from "./index";
+let inRoom = false;
+export async function startRPCIPCHandler() {
+    ipcMain.on("@rpc/page", (event, page) => {
+        if (!inRoom) {
+            switch (page) {
+                case "home":
+                    setPresence({ state: 'Exploring the home page' });
+                    break;
+                case "voice-settings":
+                    setPresence({ state: 'Customising voice settings' });
+                    break;
+                case "overlay-settings":
+                    setPresence({ state: 'Customising overlay settings' });
+                    break;
+                case "sound-effect-settings":
+                    setPresence({ state: 'Customising sound effect settings' });
+                    break;
+                default:
+                    setPresence({ state: 'Exploring the home page' });
```

Comment:
so if default is for undefined cases, I'd probably just set the state to something generic
```suggestion
                    setPresence({ state: 'Taking DogeHouse to the moon' });
```
or something like that (just a thought)

Reply:
👍